testing/conformance/src/vm.rs

```diff
@@ -268,24 +281,23 @@ impl FilecoinKernel for TestKernel {
         extra: &[u8],
     ) -> Result<Option<ConsensusFault>> {
         let charge = self
-            .0
             .price_list()
             .on_verify_consensus_fault(h1.len(), h2.len(), extra.len());
```

Comment:
Yes, we still need to access the price list on the kernel. However, the prices we're accessing here are _not_ actually "global" prices but instead filecoin-specific costs.

Reply:
hmm, not sure I understand the difference, isn't it still resolving to the same global price_list (via filecoinKernel.call_manager.price_list())? 

See  definition in Callmanager:

```rust
/// Returns the current price list.
fn price_list(&self) -> &PriceList {
    self.machine().context().price_list
}
```

where `price_list` is expanded to NetworkConfig using `derive_more` macro :

```rust
pub price_list: &'static PriceList,
```

How is this not actually the global price_list?

Reply:
Yes. My point is that, while these prices are in the "global" price list, they really shouldn't be. So this _is_ the global price list, but the "global" price list contains Filecoin-specific prices.

I'm calling this out to explain why exposing this price list on the `kernel` doesn't make that much sense, even if we're accessing it here.